tokio-macros/src/entry.rs

```diff
@@ -235,6 +277,9 @@ fn parse_knobs(
     if let Some(v) = config.worker_threads {
         rt = quote! { #rt.worker_threads(#v) };
     }
+    if let Some(v) = config.start_paused {
+        rt = quote! { #rt.start_paused(#v) };
+    }
```

Comment:
Currently this will fail to compile if used without the `test-util` feature, but making it detect that it the feature missing is somewhat cumbersome.

Reply:
Yeah - it looks like there's some detection for the multi-threaded runtime but it involves 2 macros and cfg'd imports in tokio which I don't think can easily scale to multiple features.

Reply:
I don't personally feel super strongly about supporting this in the macros, fwiw. I'd be fine leaving it out and adjusting the test to manually make the runtime.

Reply:
Can we not just do e.g.:

```
if let Some(v) = config.start_paused {
  #[cfg(not(feature="test-util"))]
  {
    return Err(syn::Error::new_spanned(v.1, "start_paused requires feature test-util"));
  }
  rt = quote! { #rt.start_paused(#v) };
}
```

Reply:
I have a memory with there being some sort of issue with forwarding the features to the tokio-macros crate (recall, it _is_ a separate crate), but I'm not sure what it was.

Reply:
Sure, I'll give it a try tonight. If it works, we could remove the `main_rt` and `test_rt` variants I think as well. Are those considered part of the crate's public API?

Reply:
Let us just keep the others for now.

Reply:
Ah - the problem with that approach is that Cargo has no way currently to say "enable this feature on an optional dependency, but *don't* activate that dependency". So if we try to propagate features into tokio-macros, turning on any of those features in tokio automatically pulls in tokio-macros.